main.py

```
@@ -49,8 +50,130 @@
+
+#Convert X_train and x_test into arrays
+X_train = X_train["features"].apply(lambda x: x.toArray() if hasattr(x, "toArray") else x).tolist()
+X_test = X_test["features"].apply(lambda x: x.toArray() if hasattr(x, "toArray") else x).tolist()
+
+# COMMAND ----------
+
+# MAGIC %run "/Workspace/Users/chris.wroe8@gmail.com/mlops/train_model"
+
+# COMMAND ----------
+
+train_model_instance  = TrainModel()
```

Ruff

Undefined name `TrainModel`

(F821)